schemes/ml

```diff
@@ -17,6 +17,13 @@ virama "~" => "്"
 
 infer_dead_consonants true
 
+word_breakers "." => ".",
+          "," => ",",
+          "?" => "?",
+          "!" => "!",
+          "(" => "(",
+          ")" => ")"
+
```

Comment:
And you don't need one in `schemes/ml` because it is not redefining any standard word break characters. It is required only in the scheme files which redefines the standard word break characters (ml-inscript for example)